api/src/services/listing.service.ts

```diff
@@ -195,11 +195,34 @@ export class ListingService implements OnModuleInit {
       }
     }
 
+    const viewInclude = views[params.view ?? 'full'];
+
+    const finalInclude = params.enableUnitGroups
```

Comment:
question: do we need to know if the unit groups are enabled? can we just always grab them and it won't find any if there aren't any unit groups? If you think it makes more sense to explicitly grab either or, can we move this logic within the `view` creation (roughly line 86 in this file)